README.md

```diff
@@ -65,6 +66,18 @@ SILENCED_SYSTEM_CHECKS = ["security.W019"]
 - Run `python manage.py collectstatic`
 - Restart your application server
 
+#### Optional features
+
+To make a fieldset start expanded with a `Hide` button to collapse:
+```python
+    fieldsets = [
+        ("Section title", {
+            "classes": ("collapse", "initial-show"),
```

Comment:
I think that a class name like `collapse-expanded` or just `expanded` would be more intuitive and easier to remember.